doc/Compilation.md

```diff
@@ -366,8 +378,8 @@ libexample/                <-- Folder name must match library name
 Like applications, libraries can leverage the Tock build system to do most of
 the heavy lifting. Simply,
 
-  1. Set `TOCK_USERLAND_BASE_DIR` to the path to the Tock userland
-  2. `include $(TOCK_USERLAND_BASE_DIR)/TockLibrary.mk`
+  1. Set `TOCK_USERLAND_BASE_DIR` to the path to the Tock userland.
+  2. `include $(TOCK_USERLAND_BASE_DIR)/TockLibrary.mk`.
```

Comment:
I feel like the period is a disservice here, looks funny and inhibits copy-paste